README.md

```diff
@@ -1,10 +1,21 @@
-[![Build Status](https://travis-ci.com/IBM/MAX-Audio-Sample-Generator.svg?branch=master)](https://travis-ci.com/IBM/MAX-Audio-Sample-Generator) [![Website Status](https://img.shields.io/website/http/max-audio-sample-generator.max.us-south.containers.appdomain.cloud/swagger.json.svg?label=api+demo)](http://max-audio-sample-generator.max.us-south.containers.appdomain.cloud/)
+[![Build Status](https://travis-ci.com/IBM/MAX-Audio-Sample-Generator.svg?branch=master)](https://travis-ci.com/IBM/MAX-Audio-Sample-Generator)
+[![Website Status](https://img.shields.io/website/http/max-audio-sample-generator.max.us-south.containers.appdomain.cloud/swagger.json.svg?label=api+demo)](http://max-audio-sample-generator.max.us-south.containers.appdomain.cloud/)
 
 # IBM Developer Model Asset Exchange: Audio Sample Generator
 
-This repository contains code to instantiate and deploy an audio generation model. The model generates short samples based on an existing dataset of audio clips. It maps the sample space of the input data and generates audio clips that are "inbetween" or "combinations" of the dominant features of the sounds. The model architecture is a generative adversarial neural network, trained by the [IBM CODAIT Team](codait.org) on lo-fi instrumental music tracks from the [Free Music Archive](http://freemusicarchive.org) and short spoken commands from the [Speech Commands Dataset](https://ai.googleblog.com/2017/08/launching-speech-commands-dataset.html). The model can generate 1.5 second audio samples of the words `up`, `down`, `left`, `right`, `stop`, `go`, as well as lo-fi instrumental music.
+This repository contains code to instantiate and deploy an audio generation model. The model generates short samples
+based on an existing dataset of audio clips. It maps the sample space of the input data and generates audio clips that
+are "inbetween" or "combinations" of the dominant features of the sounds. The model architecture is a generative
+adversarial neural network, trained by the [IBM CODAIT Team](codait.org) on lo-fi instrumental music tracks from the
```

Comment:
the link `codait.org` fails. it should be fully qualified (i.e., `http://codait.org`)